### DISCUSSION TITLE
Sony Action Cams (via card reader)

### DISCUSSION THREAD

**Discussion by sejtam** [outdoorbits/little-backup-box#211] [Q&A]
Has anyone had success with Sony Action Cams FDR_3000x, HDR-AS300, AHD-AS50 etc

I am planning to use these via a card reader, so not directly connected.

However, one comment in the Operations Wiki makes makes me a bit concerned:

`However, if files were created on the source device in the same subfolder and with the same name as files previously backed up on the target, the old files on the target will be overwritten.
`

What if I format the CD card in the camera> iirc the camera will start counting all filenames etc again at 1, including folders, thus resulting in the re-use of the files, which clashes with the above

1. can LBB format the disk in a format Sony can read/write?
2. can LBB be made to automatically delete backed up files (after a second verification run), thus obviating the need to re-format?
3. is there away to make LBB do backups strictly on a daily basis, is using the date, so as to not have clashing folders? 
3.1 I know this might end up in duplicate uploads. and that this would need LBB do know the current date, which it may need to get internet access for (can than happen via the connected phone) or would one need to manually set the date?
3.2 Is there a way to have LBB to run a de-dupe (after finishing an uplpad) to detect accidentally duplicate files (across folders) and delete/hard-link duplicates?  It may need to mark folders that are new to only have to check their files, and maintain a DB to hold the filenames, checksums etc of already known files? If possible with an intelligent dup detection
3.2.1  for duplicate detection one would not need to run the full hash, but maybe a roilling one until a divergence is found, but any new file copied would still need to be hashed, ie have the full file read)?

**Comment by outdoorbits** [comment]
If you format your cameras card and backup by cardreader (not by camera backup!) the card will get a new ID and the backup will be saved into a new folder.

**Comment by sejtam** [comment]
Sorry. not sure about this answer
a) does the card get a different ID allocated when i format it in the camera? Or only if I format it in LBB?
b) if the camera's own format does not assign an ID, i guess the id file will be missing? how are such unidentified card handles? do they then get a random ID allocated on first backup, ie be treated as different also on each post-in0camera-backup?

**Reply by outdoorbits** [reply]
1. I don't know the format Sony needs, but the have been feedback about working backup from Sony cards. But I would expect your cameras can format the card itself.

2. and 3. For safety reasons LBB never deletes source files. Even looking for duplicates would need decisions which file to delete. Comparing full files to increase security would be slow. But there is no need for that. Compare my last post.

**Reply by outdoorbits** [reply]
LBB will write an ID file, but only if the source is mounted as device. (cardreader)

**Reply by outdoorbits** [reply]
... a new ID ...

**Comment by outdoorbits** [comment]
The ID is a combination of time and a random integer.
